client/layout/guided-tours/config-elements/quit.js

```diff
@@ -12,14 +12,16 @@ import { translate } from 'i18n-calypso';
  * Internal dependencies
  */
 import Button from 'components/button';
+import { targetForSlug } from '../positioning';
 import contextTypes from '../context-types';
 
 export default class Quit extends Component {
 	static displayName = 'Quit';
 
 	static propTypes = {
+		click: PropTypes.bool,
```

Comment:
What does click mean here? Is there a more descriptive name we could have?

Reply:
Maybe `hideOnTargetClick`?

Reply:
If this was false, would target serve a purpose? Do we need this boolean?

Reply:
It's used by `Quit` for this same purpose. It seems unnecessary to me as well, but at the same time I'd avoid disrupting an established pattern

Reply:
I think it should be fine to have no `click` boolean and just check for the presence of `target`. Not a blocker though.